pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -347,21 +347,22 @@ func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 		return &SlirpBindMechanism{vmi: vmi, iface: iface, domain: domain}, nil
 	}
 	if iface.Macvtap != nil {
+		macvtapBindMech := &MacvtapBindMechanism{
+			vmi:              vmi,
+			iface:            iface,
+			domain:           domain,
+			podInterfaceName: podInterfaceName,
+		}
+
 		mac, err := retrieveMacAddress(iface)
 		if err != nil {
 			return nil, err
 		}
-		virtIface := &api.Interface{}
 		if mac != nil {
```

Comment:
We don't need this if statement anymore, since we are no longer dereferencing the "mac".

Reply:
Done.